src/api/entities/Account/types.ts

```diff
@@ -80,32 +80,32 @@ export interface HistoricPolyxTransaction extends EventIdentifier {
    * Identity from which the POLYX transaction has been initiated/deducted in case of a transfer.
    * @note this can be null in cases where some balance are endowed/transferred from treasury
    */
-  fromIdentity?: Identity;
+  fromIdentity?: Identity | undefined;
```

Comment:
Seems weird to read `?` and `| undefined`. I would have thought this would have worked. 

```suggestion
  fromIdentity: Identity | undefined;
```

From my understanding the rule is trying to stop `fromIdentity = undefined` assignment, but that shouldn't really happen, and I don't see the risk from a type perspective, consumers need to handle the undefined case regardless.